### ISSUE TITLE
 Table 11401 CBG Statement Line - procedure DeleteAppliesToID made protected

### ISSUE THREAD

**Issue by alinat2022** [microsoft/ALAppExtensions#24322]
In #19494, it was requested that procedure DeleteAppliesToID should have the accessibility level changed from local to protected.

In 22.3 version, this is still not implemented. Can you please have a look? Thank you.
